OrbitClientProtos/capture_data.proto

```diff
@@ -75,6 +74,7 @@ message CaptureInfo {
   repeated CallstackInfo callstacks = 6;
   repeated CallstackEvent callstack_events = 7;
   map<uint64, string> key_to_string = 8;
+  // TODO(kuebler): consider removing this field completely.
```

Comment:
Looks like the comment changed place accidentally

Reply:
Actually it was on purpose. As discussed with Dimitry offline, we can probably get rid of function stats for serialization completely (as this is aggregated data from timers, which are already part of the capture info proto). But in this case, we need the selected functions somehow.

Reply:
Ah, ok, I didn't know about that. Thanks for explanation!